ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -67,19 +60,9 @@
     private List<CCDCollectionElement<CCDDefendant>> defendants;
     private List<CCDCollectionElement<CCDTimelineEvent>> timeline;
     private List<CCDCollectionElement<CCDEvidenceRow>> evidence;
-
-    private CCDCountyCourtJudgment countyCourtJudgment;
-    private LocalDateTime countyCourtJudgmentRequestedAt;
     private String defendantEmail;
```

Comment:
defendantEmail is not in the mapping sheet. pls verify

Reply:
done